### PR TITLE
Added the enabled variable which controls whether the db resources are created

### PR DESCRIPTION
**Pull request by swhitaker90**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -247,3 +247,9 @@ variable "iam_database_authentication_enabled" {
   default     = false
   description = "Whether to enable IAM database authentication for the RDS Cluster"
 }
+
+variable "enabled" {
+  type        = "string"
+  default     = "1"
```
Comment (raymondbutcher) [inline variables.tf]: I much prefer `default = true` for this kind of variable.

File: variables.tf [hunk 2]
```diff
@@ -247,3 +247,9 @@ variable "iam_database_authentication_enabled" {
...
+  description = "Whether the database resources should be created"
+}
```
Comment (raymondbutcher) [inline variables.tf]: Please add a trailing newline.

File: main.tf [hunk 3]
```diff
@@ -174,6 +175,7 @@ resource "aws_db_subnet_group" "main" {
 
 // Create single DB instance
 resource "aws_rds_cluster_instance" "cluster_instance_0" {
+  count                        = "${var.enabled ? 1 : 0}"
```
Comment (raymondbutcher) [inline main.tf]: Too much whitespace.

**Comment by swhitaker90** [comment]
Hi @raymondbutcher I think I've addressed all of your comments

**Comment by raymondbutcher** [comment]
Thanks. I think these should use the new enabled variable too:
```
main.tf:
  resource "aws_rds_cluster_instance" "cluster_instance_n"
  data "aws_iam_policy_document" "monitoring-rds-assume-role-policy"

alarms.tf
  everything
```

And the readme still has `1` instead of `true`.

**Comment by swhitaker90** [comment]
@raymondbutcher Updated again
Added the variable into instance_n, the role policy & all of the alarm resources
And updated the readme

File: alarms.tf [hunk 4]
```diff
@@ -1,5 +1,5 @@
 resource "aws_cloudwatch_metric_alarm" "alarm_rds_DatabaseConnections_writer" {
-  count               = "${var.cw_alarms ? 1 : 0}"
+  count               = "${var.enabled && var.cw_alarms > 0 ? 1 : 0}"
```
Comment (raymondbutcher) [inline alarms.tf]: `> 0` here doesn't make sense when you're dealing with booleans.

```
${var.enabled && var.cw_alarms ? 1 : 0}
```

Comment (swhitaker90) [reply alarms.tf]: Removed the >0 check

File: main.tf [hunk 5]
```diff
@@ -203,7 +205,7 @@ resource "aws_rds_cluster_instance" "cluster_instance_0" {
 // Create 'n' number of additional DB instance(s) in same cluster
 resource "aws_rds_cluster_instance" "cluster_instance_n" {
   depends_on                   = ["aws_rds_cluster_instance.cluster_instance_0"]
-  count                        = "${var.replica_scale_enabled ? var.replica_scale_min : var.replica_count}"
+  count                        = "${var.enabled ? var.replica_scale_enabled ? var.replica_scale_min : var.replica_count : 0}"
```
Comment (raymondbutcher) [inline main.tf]: I've never seen this before but it seems like it'll work.

**Pull request merged**